diff -uprN old/kernel/timeconst.pl new/kernel/timeconst.pl
--- old/kernel/timeconst.pl	2008-12-25 07:26:37.000000000 +0800
+++ new/kernel/timeconst.pl	2020-05-23 12:01:11.713467743 +0800
@@ -370,7 +370,7 @@ if ($hz eq '--can') {
 	}
 
 	@val = @{$canned_values{$hz}};
-	if (!defined(@val)) {
+	if (!(@val)) {
 		@val = compute_values($hz);
 	}
 	output($hz, @val);
diff -uprN old/Makefile new/Makefile
--- old/Makefile	2008-12-25 07:26:37.000000000 +0800
+++ new/Makefile	2020-05-23 12:01:05.201389163 +0800
@@ -439,7 +439,7 @@ ifeq ($(config-targets),1)
 include $(srctree)/arch/$(SRCARCH)/Makefile
 export KBUILD_DEFCONFIG KBUILD_KCONFIG
 
-config %config: scripts_basic outputmakefile FORCE
+%config: scripts_basic outputmakefile FORCE
 	$(Q)mkdir -p include/linux include/config
 	$(Q)$(MAKE) $(build)=scripts/kconfig $@
 
@@ -1604,7 +1604,7 @@ endif
 	$(Q)$(MAKE) $(build)=$(build-dir) $(target-dir)$(notdir $@)
 
 # Modules
-/ %/: prepare scripts FORCE
+%/: prepare scripts FORCE
 	$(cmd_crmodverdir)
 	$(Q)$(MAKE) KBUILD_MODULES=$(if $(CONFIG_MODULES),1) \
 	$(build)=$(build-dir)
