diff -uprN old/Makefile new/Makefile
--- old/Makefile	2006-03-20 13:53:29.000000000 +0800
+++ new/Makefile	2020-05-23 09:52:06.145616228 +0800
@@ -438,7 +438,7 @@ ifeq ($(config-targets),1)
 include $(srctree)/arch/$(ARCH)/Makefile
 export KBUILD_DEFCONFIG
 
-config %config: scripts_basic outputmakefile FORCE
+%config: scripts_basic outputmakefile FORCE
 	$(Q)mkdir -p include/linux
 	$(Q)$(MAKE) $(build)=scripts/kconfig $@
 	$(Q)$(MAKE) -C $(srctree) KBUILD_SRC= .kernelrelease
