diff -uprN old/include/configs/vexpress_common.h new/include/configs/vexpress_common.h
--- old/include/configs/vexpress_common.h	2014-07-15 01:16:45.000000000 +0800
+++ new/include/configs/vexpress_common.h	2020-02-29 09:58:36.508636560 +0800
@@ -219,6 +219,8 @@
 		"ramdisk_addr=0x44800000\0" \
 		"maxramdisk=0x1800000\0" \
 		"pxefile_addr_r=0x88000000\0" \
+		"BiscuitOS_kernel=0x60200000\0" \
+		"BiscuitOS_DTB=0x60800000\0" \
 		"kernel_addr_r=0x80008000\0"
 #elif defined(CONFIG_VEXPRESS_EXTENDED_MEMORY_MAP)
 #define CONFIG_PLATFORM_ENV_SETTINGS \
@@ -240,9 +242,17 @@
 		"flashargs=setenv bootargs root=${root} console=${console} " \
 			"mem=${dram} mtdparts=${mtd} mmci.fmax=190000 " \
 			"devtmpfs.mount=0  vmalloc=256M\0" \
-		"bootflash=run flashargs; " \
-			"cp ${ramdisk_addr} ${ramdisk_addr_r} ${maxramdisk}; " \
-			"bootm ${kernel_addr} ${ramdisk_addr_r}\0"
+		"bootflash=echo ' '; " \
+			"echo ' ____  _                _ _    ___  ____  '; " \
+			"echo '| __ )(_)___  ___ _   _(_) |_ / _  / ___| '; " \
+			"echo '|  _  | / __|/ __| | | | | __| | | ___   '; " \
+			"echo '| |_) |  __   (__| |_| | | |_| |_| |___) |'; " \
+			"echo '|____/|_|___/ ___| __,_|_| __| ___/|____/ '; " \
+			"echo ' '; " \
+			"fatload mmc 0:1 ${BiscuitOS_kernel} uImage; " \
+			"fatload mmc 0:1 ${BiscuitOS_DTB} vexpress-v2p-ca9.dtb; " \
+			"setenv bootargs earlycon root=/dev/mmcblk0p2 rw rootfstype=ext4 console=ttyAMA0 init=/linuxrc loglevel=8; " \
+			"bootm ${BiscuitOS_kernel} - ${BiscuitOS_DTB}\0"
 
 /* FLASH and environment organization */
 #define PHYS_FLASH_SIZE			0x04000000	/* 64MB */
@@ -288,7 +298,8 @@
 
 /* Monitor Command Prompt */
 #define CONFIG_SYS_CBSIZE		512	/* Console I/O Buffer Size */
-#define CONFIG_SYS_PROMPT		"VExpress# "
+#define CONFIG_SYS_PROMPT		"BiscuitOS=> "
+#define CONFIG_CMD_ECHO			1
 #define CONFIG_SYS_PBSIZE		(CONFIG_SYS_CBSIZE + \
 					sizeof(CONFIG_SYS_PROMPT) + 16)
 #define CONFIG_SYS_HUSH_PARSER
