From 8111488d9b8afa59572051c491f571c68ded0eba Mon Sep 17 00:00:00 2001
From: BuddyZhang1 <buddy.zhang@aliyun.com>
Date: Tue, 24 Jul 2018 11:02:17 +0800
Subject: [PATCH 3/3] remove some information

---
 init/main.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/init/main.c b/init/main.c
index 5cfe549..a7f19bc 100644
--- a/init/main.c
+++ b/init/main.c
@@ -249,9 +249,9 @@ void init(void)
     (void)open("/dev/tty0", O_RDWR, 0);
     (void)dup(0);
     (void)dup(0);
-    printf("%d buffers = %d bytes buffer space\n\r", NR_BUFFERS,
-            NR_BUFFERS * BLOCK_SIZE);
-    printf("Free mem: %d bytes\n\r", memory_end - main_memory_start);
+//    printf("%d buffers = %d bytes buffer space\n\r", NR_BUFFERS,
+  //          NR_BUFFERS * BLOCK_SIZE);
+//    printf("Free mem: %d bytes\n\r", memory_end - main_memory_start);
 #ifdef CONFIG_DEBUG_USERLAND_SYSCALL
     debug_on_userland_syscall();
 #endif
-- 
1.9.1

